samples/cellular/modem_shell/src/shell.c

```diff
@@ -219,23 +214,10 @@ int heap_shell(const struct shell *shell, size_t argc, char **argv)
 		mosh_print("allocated:      %6d", kernel_stats.allocated_bytes);
 		mosh_print("max. allocated: %6d\n", kernel_stats.max_allocated_bytes);
 	}
-#endif /* CONFIG_SYS_HEAP_RUNTIME_STATS */
-
-	/* Calculate the system heap maximum size. */
-#define USED_RAM_END_ADDR POINTER_TO_UINT(&_end)
-#define HEAP_BASE USED_RAM_END_ADDR
-#define MAX_HEAP_SIZE (KB(CONFIG_SRAM_SIZE) - (HEAP_BASE - CONFIG_SRAM_BASE_ADDRESS))
-
-	system_stats = mallinfo();
-
-	mosh_print("system heap statistics:");
-	mosh_print("max. size:      %6d", MAX_HEAP_SIZE);
-	mosh_print("size:           %6d", system_stats.arena);
-	mosh_print("free:           %6d", system_stats.fordblks);
-	mosh_print("allocated:      %6d", system_stats.uordblks);
 
```

Comment:
why this was done? so, system heap statistics are not printed anymore within mosh heap command?

Reply:
ah, NCSDK-33478